lib/inputstreamhelper/__init__.py

```diff
@@ -959,3 +996,23 @@ def info_dialog(self):
         text += localize(30830, url=config.ISSUE_URL)  # Report issues
 
         Dialog().textviewer(localize(30901), text)
+
+    def rollback_libwv(self):
+        """Backup current lib and rollback to a version specified by the user"""
+        backup_path = self._backup_path()
+        versions = os.listdir(backup_path)
```

Comment:
Not sure if we should remove the current installed version from the version to rollback to. Is there a use case to rollback to the current version?

Reply:
Not really. As long as the backups are copied to the cdm dir and not linked, it might be useful if the installed version was somehow corrupted. When we switch to links, the only reason I can think of is when you decrease the number of backups this would call `remove_old_backups()` quicker than doing a reinstallation.
This reminds me that I forgot to add a check to `remove_old_backups()`, so it wouldn't remove the installed version. This wouldn't necessarily be a problem at the moment, but definitely when we change to links.

Reply:
I added the check to `remove_old_backups()` and I removed the installed version.